<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOListWidget</class>
 <widget class="QWidget" name="IOListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="top_lay">
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QLabel" name="io_name_lbl">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QRadioButton" name="toggle_rbtn">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="extremes_lay">
     <item>
      <layout class="QHBoxLayout" name="lower_extr_lay">
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="lower_descr_lbl">
         <property name="text">
          <string>Lower:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="lower_dbox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="upper_extr_lay">
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="upper_descr_lbl">
         <property name="text">
          <string>Upper:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="upper_dbox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
